### DISCUSSION TITLE
Get animated stickers for Steam Scream Fest 2024

### DISCUSSION THREAD

**Discussion by lzrdblzzrd** [chr233/ASFEnhance#332] [杂谈 / General]
`RPI ASF 334213, 334214, 334215, 334216, 334217, 334218, 334219`
---
![store steampowered com_points_shop_app_3257180_cluster_6](https://github.com/user-attachments/assets/0973f4d2-2094-4a77-bc2d-5690de3dc5a5)

**Comment by woctezuma** [comment]
Be careful, as I think the 7 stickers are the only **free** items now. That was not the case when the Fest started.

https://store.steampowered.com/points/shop/app/3257180/cluster/6

![qmdOjTSF3H](https://github.com/user-attachments/assets/8a1c13a1-39cf-48cf-ba98-53c6274c6c08)

**Reply by lzrdblzzrd** [reply]
@woctezuma Edited my comment
